FAILED: FlowExpandSpec

~~~scala
[info] Expand
[info] - must pass-through elements unchanged when there is no rate difference (42 milliseconds)
[info] - must expand elements while upstream is silent (539 milliseconds)
[info] - must do not drop last element (528 milliseconds)
[info] - must work on a variable rate chain (671 milliseconds)
[info] - must backpressure publisher when subscriber is slower (1 second, 32 milliseconds)
[info] - must work properly with finite extrapolations *** FAILED *** (341 milliseconds)
[info]   java.lang.AssertionError: assertion failed: expected OnComplete, found OnNext((2,1))
[info]   at scala.Predef$.assert(Predef.scala:165)
[info]   at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:351)
[info]   at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:327)
[info]   at akka.testkit.TestKit.expectMsg(TestKit.scala:735)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectComplete(StreamTestKit.scala:346)
[info]   at akka.stream.scaladsl.FlowExpandSpec$$anonfun$1$$anonfun$apply$mcV$sp$6.apply$mcV$sp(FlowExpandSpec.scala:146)
[info]   at akka.stream.scaladsl.FlowExpandSpec$$anonfun$1$$anonfun$apply$mcV$sp$6.apply(FlowExpandSpec.scala:129)
[info]   at akka.stream.scaladsl.FlowExpandSpec$$anonfun$1$$anonfun$apply$mcV$sp$6.apply(FlowExpandSpec.scala:129)
~~~

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowExpandSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowExpandSpec.scala
@@ -135,9 +135,9 @@ class FlowExpandSpec extends AkkaSpec {
       source
         .sendNext(1)
       sink
-        .request(5)
+        .request(4)
         .expectNext(1 -> 0, 1 -> 1, 1 -> 2)
-        .expectNoMsg(300.millis)
+        .expectNoMsg(100.millis)
       source
         .sendNext(2)
         .sendComplete()

